lib/mongo/collection.rb

```diff
@@ -239,6 +239,12 @@ def create(opts = {})
       operation.delete(:write)
       operation.delete(:write_concern)
       client.send(:with_session, opts) do |session|
+        temp = write_concern
+        write_concern = if opts[:write_concern]
+          WriteConcern.get(opts[:write_concern])
+        else
+          temp
```

Comment:
You should be able to replace this with `self.write_concern` and get rid of line 242.

Reply:
For this and the other methods, can you please add write_concern as an accepted option to the docstring.